src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -70,7 +70,7 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
      * Otherwise, any error will make Lorre proceed as expected by default (stop or wait for next cycle)
      */
     val attemptedProcessing =
-      if (sys.env.get("LORRE_FAILURE_IGNORE").forall(ignore => ignore == "false" || ignore == "no"))
+      if (sys.env.get("LORRE_FAILURE_IGNORE").forall(ignore => ignore == "false" || ignore == "no") || didConnect)
```

Comment:
I'm not sure this does what it's needed...
This flag should control wether Lorre stops when processing is failed with one of 
`AccountsProcessingFailed` or `BlocksProcessingFailed`, or if they should be actually ignored...
But this branch of flow should not be the one commonly used anyway... hence probably the connection failures pass through anyway and will stop the process even when didConnect is false

Reply:
Besides what @ivanopagano said, it seems to me the 'boot up' behavior is distinct from the 'run time' behavior, i.e. it might make sense to have a distinct function which is executed at Lorre start up which 'blocks' until the Tezos host is contactable.

Reply:
@vishakh: you mean some kind of pre-start sequence, explicitly denoted separately from the main loop?

Reply:
@ivanopagano I've tested it - behaviour I've implemented do not fail lorre until it successfully finishes processing, connection failures do not cause lorre to fail. 

But from what @vishakh said I need to implement it in some way that checks connection to the tezos node. And here I need your opinion on how to do it. I've looked into the documentation https://tezos.gitlab.io/alphanet/api/rpc.html#rpc-index and I'm not sure on which one I can rely that gives me current status of the node - if it is possible to be queried. I could go for the `blocks`, but it would be doubling the the query. Or do we assume that if any endpoint works every other works as well?

Reply:
Ok, I've changed the implementation to run blocking check for blocks head before the main loop recursively so until connection to the tezos node will be established, the `mainLoop` won't start. Added new config value for retry interval.